waku/v2/protocol/waku_rln_relay/waku_rln_relay_metrics.nim

```diff
@@ -0,0 +1,30 @@
+import
+  metrics,
+  std/times
+
+export metrics
+
+declarePublicCounter(waku_rln_messages, "number of messages published on the rln content topic")
+declarePublicCounter(waku_rln_spam_messages, "number of spam messages detected")
+declarePublicCounter(waku_rln_invalid_messages, "number of invalid messages detected", ["type"])
+# This metric will be useful in detecting the index of the root in the acceptable window of roots
+declarePublicCounter(waku_rln_valid_messages, "number of valid messages with their roots tracked", ["index"])
+declarePublicCounter(waku_rln_errors, "number of errors detected while operating the rln relay", ["type"])
+declarePublicCounter(waku_rln_proof_verification, "number of times the rln proofs are verified")
```

Comment:
Let's follow Prometheus' best practices when defining metric names: http_requests_total (for a unit-less accumulating count)

I will have to do it in waku store sooner than later. But now is the right moment for the RLN relay 👍🏼

Reply:
Addressed in [2a32c82](https://github.com/status-im/nwaku/pull/1185/commits/2a32c82d6e68db06514570c6939afe70bd59fa93)